tools/tasks/seed/compile.ahead.prod.ts

```diff
@@ -9,64 +10,64 @@ import { argv } from 'yargs';
 
-function codegen(
-  ngOptions: AngularCompilerOptions,
-  cliOptions: NgcCliOptions,
-  program: ts.Program,
-  host: ts.CompilerHost
-) {
-  return CodeGenerator.create(ngOptions, cliOptions, program, host).codegen();
-}
+const NgCompilerOptions = ['lang'];
 
 const modifyFile = (path: string, mod: any = (f: string) => f) => {
   const file = readFileSync(path);
   writeFileSync(path, mod(file.toString()));
 };
 
+const updateCompilerOptions = (options: any, task: string) => {
+  options.declaration = true;
+  if (task.indexOf('rollup') >= 0) {
+    options.module = 'es2015';
+    options.moduleResolution = 'node';
+  }
+};
+
 export = (done: any) => {
   // Note: dirty hack until we're able to set config easier
   modifyFile(join(Config.TMP_DIR, 'tsconfig.json'), (content: string) => {
     const parsed = JSON.parse(content);
-    const path = join(
-      Config.PROJECT_ROOT,
-      Config.TOOLS_DIR,
-      'manual_typings',
-      'project'
-    );
+    const path = join(Config.PROJECT_ROOT, Config.TOOLS_DIR, 'manual_typings', 'project');
+    updateCompilerOptions(parsed.compilerOptions, argv._[0]);
     parsed.files = parsed.files || [];
     parsed.files = parsed.files.concat(
       readdirSync(path)
         .filter(f => f.endsWith('d.ts'))
         .map(f => join(path, f))
     );
-    parsed.files = parsed.files.filter(
-      (f: string, i: number) => parsed.files.indexOf(f) === i
-    );
+    parsed.files = parsed.files.filter((f: string, i: number) => parsed.files.indexOf(f) === i);
     parsed.files.push(join(Config.BOOTSTRAP_DIR, 'main.ts'));
     return JSON.stringify(parsed, null, 2);
   });
-  const args = argv as any;
+  const args: string[] = [];
+  const namedArgs = argv;
 
-  // If a translation, tell the compiler
-  if (args.lang) {
-    let i18nFilePath = `${Config.LOCALE_DEST}/messages.${args.lang}.xlf`;
-    let isExists = existsSync(i18nFilePath);
-    if (isExists) {
-      args['i18nFile'] = i18nFilePath;
-      args['locale'] = args.lang;
-      args['i18nFormat'] = 'xlf';
-    } else {
-      util.log(util.colors.gray('Translation file is not found'), util.colors.yellow(i18nFilePath));
-      util.log(util.colors.gray(`Use 'npm run i18n' command to create your translation file`));
-    }
-  }
-
-  const cliOptions = new NgcCliOptions(args);
-  main(Config.TMP_DIR, cliOptions, codegen)
-    .then(done)
-    .catch((e: Error) => {
-      console.error(e.stack);
-      console.error('Compilation failed');
-      process.exit(1);
+  args.push('-p', join(Config.TMP_DIR, 'tsconfig.json'));
+  Object.keys(namedArgs)
+    .filter(n => NgCompilerOptions.indexOf(n) >= 0)
+    .forEach((key: string) => {
+      if (key === 'lang') {
+        const lang: string = namedArgs[key] as string;
+        const i18nFilePath = `${Config.LOCALE_DEST}/messages.${lang}.xlf`;
+        const isExists = existsSync(i18nFilePath);
+        if (isExists) {
+          args.push('--i18n-file', i18nFilePath);
```

Comment:
I think this needs to be: `args.push('--i18nFile', i18nFilePath);` as it has changed from --i18n-file.
https://angular.io/guide/i18n#merge-the-completed-translation-file-into-the-app

Reply:
Thanks! You tested it with `--i18nFilePath` and it works properly?

Reply:
I ran it with `npm run build.prod.aot -- --lang en-US` with the above change and it works properly. I'm not sure about `--i18nFilePath`  but `--i18nFile` , yes that works.